### PR TITLE
Add draft of Dask Summit blogpost

### PR DESCRIPTION
**Pull request by mrocklin**
We would like to write up content from the Dask summit in February.  

So far, I've pushed a skeleton structure.  Critical feedback would be welcome.

cc @mmccarty I've given you access to my fork so you should be able to push to this branch

### CODE REVIEW

**Comment by mmccarty** [comment]
Final thoughts section looks good. I went through and listed some common threads among the issues in case you wanted to discuss any of them. Some of you touch on.

-   Conda environments
-   Deployment, cluster, and computation management
-   Documentation discussing naratives for HPC users
    -   best practices, debugging, controlling workers, dashboard access, centralized management of multiple Dask clusters
-   Heterogenious workers
-   Smarter scheduler (task retry, graph optimization strategies)
-   HA recovery and resiliency for both the scheduler and workers

**Comment by mrocklin** [comment]
I think that adding a list of topics like that, perhaps ended with "among many others..." makes sense to me.

File: _posts/2020-04-17-dask-summit.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+-   SymphonyRM
+-   The HDF Group
+-   USGS
+-   Ursa Labs
+
+
+Objectives
+----------
+
+The Dask community comes from a broad range of backgrounds.  .
```
Comment (stsievert) [inline _posts/2020-04-17-dask-summit.md]: Typo: extra period Suggested change:

``` diff
- ... range of backgrounds.  .
+ ... range of backgrounds.
```

File: _posts/2020-04-17-dask-summit.md [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+-   Presenters: Jonathan MacCarthy
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vSWAgKLxt1tBZxXjQfIRQNFPvAMFYZ-z0hkMy7euPnOHwO9pomH_gM8cKUTKXA68w/embed?start=false&loop=false&delayms=3000" frameborder="0" width="600" height="404" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+
+Unstructured Time
+-----------------
+
+Having rapid fire talks in the morning, followed by unstructured time in the
+afternoon was a productive combination that we would recommend to other broad
+groups in the future.  Engagement and productivity was really high throughout
```
Comment (stsievert) [inline _posts/2020-04-17-dask-summit.md]: > followed by unstructured time in the
afternoon was a productive combination that we would recommend to other broad
groups in the future. Engagement and productivity was really high throughout...

My first thought when reading this paragraph: what happened in the unstructured time? I want to know why unstructured time is useful before the recommendation.   Maybe this reorganization would be slightly better:

> Having rapid fire talks in the morning, followed by unstructured time in the afternoon was a productive combination. Below you'll see pictures from geo-scientists and quants talking about the same challenges, and library maintainers from Pandas/Arrow/RAPDIS/Dask all working together on joint solutions.
>
> [images]
>
> This unstructured time is a productive combination that we would recommend to other technically diverse groups in the future. Engagement and productivity was really high throughout the workshop.

File: _posts/2020-04-17-dask-summit.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vSk2zAnSmzpbz5BgK70mpPmeQeV4h1IkCQh-EU8KXrZFJQGHmlMTuHvln3CmOQVTg/embed?start=false&loop=false&delayms=3000" frameborder="0" width="600" height="360" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+
+#### Prefect
+
+-   Title: Prefect + Dask: Parallel / Distributed Workflows
+-   Presenters: Chris White, CTO
+
+<iframe src="//www.slideshare.net/slideshow/embed_code/key/4wiUwkDHmdzVTW" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/ChrisWhite249/dask-prefect" title="Dask + Prefect" target="_blank">Dask + Prefect</a> </strong> from <strong><a href="https://www.slideshare.net/ChrisWhite249" target="_blank">Chris White</a></strong> </div>
```
Comment (gforsyth) [inline _posts/2020-04-17-dask-summit.md:139]:
```suggestion
<iframe src="//www.slideshare.net/slideshow/embed_code/key/4wiUwkDHmdzVTW" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/ChrisWhite249/dask-prefect" title="Dask + Prefect" target="_blank">Dask + Prefect</a> </strong> from <strong><a href="https://www.slideshare.net/ChrisWhite249" target="_blank">Chris White</a></strong> </div></iframe>
```

File: _posts/2020-04-17-dask-summit.md (hunk 1 above)
Comment (mrocklin) [reply _posts/2020-04-17-dask-summit.md]: Done.  Thanks @stsievert !

File: _posts/2020-04-17-dask-summit.md (hunk 2 above)
Comment (mrocklin) [reply _posts/2020-04-17-dask-summit.md]: Done.  Thanks @stsievert !

**Comment by mrocklin** [comment]
Whoops.  It looks like I had a lingering commit locally with the changes from @stsievert .  Pushed.

It seems like we've converged here.  Any objection if I pull the trigger tomorrow?

**Comment by mmccarty** [comment]
@stsievert - Thank you for the feedback! 
@mrocklin - No objection here. Thanks!

**Comment by stsievert** [comment]
No objections here. I’m looking forward to sharing this post!

**Pull request merged**